### PR TITLE
Fix/language update suggestions to basic css basic html

### PR DESCRIPTION
**Pull request by schalkneethling**
#### Pre-Submission Checklist

- [x] Your pull request targets the `staging` branch of freeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](http://forum.freecodecamp.org/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [x] All new and existing tests pass the command `npm test`. Use `git commit --amend` to amend any fixes.

#### Type of Change

- [x] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)

#### Checklist:

- [x] Tested changes locally.
- [ ] Addressed currently open issue (replace XXXXX with an issue no in next line)

Closes #XXXXX

#### Description
Some additional updates to the language for basic-html as well as basic-css. I also changed all instances of `Monospace`, `Sans-Serif`, and `Serif` to be all lower case(in basic-css). Even though these are not case sensitive, it is a widely used common practice to specify these fallbacks using all lower case.

see: https://developer.mozilla.org/en-US/docs/Web/CSS/font-family for example

### CODE REVIEW

File: seed/challenges/01-responsive-web-design/basic-css.json [hunk 1]
```diff
@@ -696,15 +696,16 @@
       "id": "bad87fee1348bd9aedf08807",
       "title": "Import a Google Font",
       "description": [
-        "Now, let's import and apply a Google font (note that if Google is blocked in your country, you will need to skip this challenge).",
+        "In addition to specifying common fonts that are found on most operating systems, we can also specify non-standard, custom web fonts for use on our website. There are varies type foundries on the web but, for this example we will focus on the Google Fonts library.",
```
Comment (Bouncey) [inline seed/challenges/01-responsive-web-design/basic-css.json]: ~There are varies type foundries on the web but,~

There are various sources for web fonts on the internet but,

**Review by Bouncey — COMMENTED** [review]
We have to remember that not all users are native English speakers, So we have to try and keep the language plain.

**Comment by schalkneethling** [comment]
Thanks for the review @Bouncey - updated.

**Review by Bouncey — APPROVED** [review]

**Comment by Bouncey** [comment]
Thanks @schalkneethling 👍

**Pull request merged**